app/models/manageiq/providers/redhat/infra_manager/ovirt_services/strategies/v4.rb

```diff
@@ -405,6 +405,19 @@ def update_cloud_init!(content)
           )
         )
       end
+
+      def platform
+        vm = get
+        vm.os.type
+      end
```

Comment:
I wonder if we have a better way to determine if a virtual machine has Windows as the operating system. ManageIQ does extract that information from the virtual machine, and also performs smart analysis. Isn't there a way to use that information in order to determine if the OS is Windows in a more reliable way? I mean, the `os.type` may change, it isn't part of the contract of the oVirt API, it comes from configuration files in the engine. Tomorrow it could be `win` instead of `windows` (very unlikely) and then this would not work.

Reply:
Also I'd suggest to repeat the `windows` string, make it a constant, or a configuration setting.

Reply:
Good point, I will explore it and make `windows` a constant.